src/scripts/main.js

```diff
@@ -5,4 +5,36 @@ const people = require('./lib/people');
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const result = document.querySelector('.dashboard');
+const arr = JSON.parse(JSON.stringify(people));
```

Comment:
here the same, try to use more meaningful names :) peopleArray, for example

Reply:
ok, i changed them for cell, but didn`t pay attention to the rest) thanks, it`s really my weak side